### ISSUE TITLE
projects need gitignore to avoid too much space

### ISSUE THREAD

**Issue by mu-us61** [Kilo-Loco/30DaysOfFlutter#1]
hello, firstly thanks for this projects 
projects need gitignore to avoid too much space , like builds folder etc.

i checked flutter has offical one ; https://github.com/flutter/flutter/blob/master/.gitignore

if you download git code it takes 1.9 GB zip file , it is so much, some projects are in 1 GB size
(project has a custom made gitignore file but does not enough filtering)

![image](https://user-images.githubusercontent.com/73366454/119281717-5e8a1000-bc3f-11eb-8ada-892d929dde16.png)

**Comment by yurydemin** [comment]
> 
> 
> hello, firstly thanks for this projects
> projects need gitignore to avoid too much space , like builds folder etc.
> 
> i checked flutter has offical one ; https://github.com/flutter/flutter/blob/master/.gitignore
> 
> if you download git code it takes 1.9 GB zip file , it is so much, some projects are in 1 GB size
> (project has a custom made gitignore file but does not enough filtering)
> 
> ![image](https://user-images.githubusercontent.com/73366454/119281717-5e8a1000-bc3f-11eb-8ada-892d929dde16.png)

Yep, they have shared extended gitignore, musthave! Thanks for that
